customer_member_card/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+#coding:utf-8
+{
+    'name': 'customer_member_card',
+
+    'author': "Polln group",
+
+    'summary': """
+    Module that customizes the base module and contains some python tools
+     """,
+
+    'website': 'https://github.com/beescoop/Obeesdoo',
+
+    'category': 'Sales',
+
+    'version': '12.0.1.0.1',
+
+    'depends': ['base'],
+
+    'data': ['views/member_card_logo.xml'],
+
+    'license': 'AGPL-3',
+
+    'installable': True,
+
+    'application': True,
```

Comment:
As for the other PRs, this is not an application. Can you also remove the empty lines ?